test/runtests.jl

```diff
@@ -9,39 +9,25 @@ const TRIXI_MPI_NPROCS = clamp(Sys.CPU_THREADS, 2, 3)
 @time @testset "Trixi.jl tests" begin
   @time if TRIXI_TEST == "all" || TRIXI_TEST == "1D"
     include("test_examples_1d.jl")
-    include("test_examples_1d_old.jl") # TODO: Taal remove
   end
 
   @time if TRIXI_TEST == "all" || TRIXI_TEST == "2D"
     include("test_examples_2d.jl")
   end
 
-  @time if TRIXI_TEST == "all" || TRIXI_TEST == "2D_OLD" # TODO: Taal remove
-    include("test_examples_2d_old.jl")
-  end
-
   @time if (TRIXI_TEST == "all" && !ON_APPVEYOR) || TRIXI_TEST == "3D"
     include("test_examples_3d.jl")
   end
 
-  @time if (TRIXI_TEST == "all" && !ON_APPVEYOR) || TRIXI_TEST == "3D_OLD" # TODO: Taal remove
-    include("test_examples_3d_old.jl")
-  end
-
   @time if TRIXI_TEST == "all" || TRIXI_TEST == "misc"
     include("test_manual.jl")
     include("test_special_elixirs.jl")
-    include("test_special_elixirs_old.jl") # TODO: Taal remove
   end
 
   @time if (TRIXI_TEST == "all" && !ON_APPVEYOR) || TRIXI_TEST == "paper-self-gravitating-gas-dynamics"
```

Comment:
Didn't we get rid of AppVeyor testing? If yes, we should also remove these `ON_APPVEYOR` variables

Reply:
You're right, thanks!

Reply:
Done in 04d19af.